Bio/PDB/vectors.py

```diff
@@ -281,6 +281,8 @@ def __div__(self, x):
         a = self._ar / numpy.array(x)
         return Vector(a)
 
+    __truediv__ = __div__
```

Comment:
What if we do the opposite? Rename the current method to `__truediv__` and assign `__div__ = __truediv__`. That way, when we drop 2.x support, we can just remove `__div__`.

Reply:
That seems a worthwhile improvement.